apps/Dataman/src/screens/Location.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState, useEffect } from "react";
+import { Text, View } from "react-native";
+import * as Location from "expo-location";
+
+export const LocationScreen = () => {
+  const [errorMsg, setErrorMsg] = useState("");
+  const [locationServiceEnabled, setLocationServiceEnabled] = useState(false);
+  const [displayCurrentAddress, setDisplayCurrentAddress] = useState(
+    "Wait, we are fetching you location..."
+  );
+
+  useEffect(() => {
+    CheckIfLocationEnabled();
+    GetCurrentLocation();
+  }, []);
```

Comment:
empty dependency array is a bad practice

Reply:
now I have given the array dependency when the button is clicked the state will change.